Nautilus/Documentation/tutorials/vehicle-module.md

```diff
@@ -0,0 +1,137 @@
+# Using the Vehicle Upgrade Module Gadget
+
+Vehicles modules are often hard to mod, and when multiple mods patch a function of a vehicle it can make severe conflicts leading to one of both mods to not work.  
```

Comment:
"leading to one of both mods to not work" sounds a bit unnatural, maybe change it to something like "which can cause one or both mods to break"

Reply:
I'm changing that